### PR TITLE
fix many typos

### PR DESCRIPTION
**Pull request by fchapoton**
most of them found using codespell ;

and there still remains many others

### CODE REVIEW

File: EXAMPLES/cubes/makeCube.cpp [hunk 1]
```diff
@@ -31,9 +31,9 @@ struct QuickFraction
 
 /**
  * Makes a cube, [0, scale]^n, as a latte facet file.
- * @parm filename: output file name
- * @parm dim: dim = n from above.
- * @parm scale: we scale the unit cube.
+ * @param filename: output file name
+ * @param dim: dim = n from above.
+ * @param scale: we scale the unit cube.
  */
```
Comment (mkoeppe) [inline EXAMPLES/cubes/makeCube.cpp:37]: I think this was intended to be a doxygen keyword, shouldn't be changed

**Comment by fchapoton** [comment]
this was for consistency with all other files. But apparently, I chose the wrong keyword

Indeed, doxygen has `@param` but not `@parm`

https://www.doxygen.nl/manual/commands.html#cmdparam

**Review by mkoeppe — APPROVED** [review]

**Comment by fchapoton** [comment]
so, ready to merge ?

**Comment by mkoeppe** [comment]
Thanks!

**Pull request merged**